cypress/Objects/TimeLoggin.js

```diff
@@ -0,0 +1,28 @@
+class TimeLoggin{
+    urlIncludeTimeLogging(){
+        return cy.url()
+    }
+    getPageTitle(){
+        return cy.get('.page__title')
+    }
+    getCalendar(){
+        return cy.get('.calendar')
+    }
+    getTimeTile(){
+        return cy.get('.tile.form')
+    }
+    getUserInfoFromTitle(){
```

Comment:
Methods like these could be put in `common.js` or `pageComponents.js` files, because this is shared between pages and if you would want to check username it would look weird if you have to use `timeLogging `to do so.